docs/tutorials/02-channel-upgrades/05-upgrade-channel.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Upgrade channel
+sidebar_label: Upgrade channel
+sidebar_position: 5
+slug: /channel-upgrades/upgrade-channel
+---
+
+# Upgrade the ICS 20 transfer channel
+
+## Start the relayer
+
+We start the relayer:
+
+```bash
+hermes --config config.toml start
+```
+
+## Initiate the upgrade
```

Comment:
maybe we can instead refer people to the cli added for this purpose [here](https://github.com/cosmos/ibc-go/blob/f9b918f13ec5c242927e273179ca2f160c56e19b/modules/core/04-channel/client/cli/tx.go#L61-L150)